Suggested words to clarify different roles

meiqimichelle opened the pull request:
I'm not sure if these are the exact words, or the exact approach, but we do need to clarify what we're looking for in each role. There is too much overlap as it stands, and the "Front End Designer key objectives" don't include any design objectives, which doesn't seem right given discussion happening elsewhere.

/cc @shawnbot @msecret @maya @afeld

meiqimichelle committed:
Suggested words to clarify different roles

I'm not sure if these are the exact words, or the exact approach, but we do need to clarify what we're looking for in each role. There is too much overlap as it stands, and the "Front End Designer key objectives" don't include any design objectives, which doesn't seem right given discussion happening elsewhere.

/cc @shawnbot @msecret @maya @afeld

meiqimichelle commented:
Agree. I think it would be better to clarify the positions directly, but am guessing that will take some time. Would love several eyes/opinions on whether what I wrote accurately reflects what we've been honing in on as the difference between front end folks on Team Design versus Team Engineering.

msecret commented on pages/who-we-are-hiring.md:
```diff
@@ -18,6 +18,12 @@ We're actively hiring for the following positions:
+
+We expect our Front End Designers to have design as well as HTML, CSS and some JavaScript experience. They should feel comfortable brainstorming and developing design options to meet user needs; developing wireframes or HTML/CSS prototypes to test their design ideas; and quickly iterating on feedback. They should also feel comfortable working as a front end developer on projects, implementing acc ...
+
+By contrast, our Front End Web Developers are not expected to have design experience. Instead, they are expected to have much more experience than our Front End Designers with JavaScript, including strict mode compliance, modularization techniques and tools, and frameworks and libraries. They are expected to be fully comfortable writing vanilla JavaScript and working with APIs.
```
I'd technically like to have something about "paying attention to front end performance" here but it's not that important for me.

msecret commented:
I agree with you guys, we should eventually clarify the two separate positions but this quick summary on whats the difference is really useful right now.

meiqimichelle committed:
Adds language on performance

@msecret whatcha think?

maya commented:
I think this will be very useful too. 

Qq: what do you mean by "modularization techniques" in the dev description? @meiqimichelle 

I think it's just as important to somehow explain where these two positions overlap. Doesn't have to be here but somehow.

meiqimichelle commented:
@maya I don't know, it's in the position description. Since I wasn't sure what Front End Dev on engineering side wants from their folks, I mostly just copied from their open position :-/

maya commented:
@meiqimichelle Ok bc if it's referring to writing modular CSS, I'd expect that from both.

russu commented on pages/who-we-are-hiring.md:
```diff
@@ -18,6 +18,12 @@ We're actively hiring for the following positions:
 
+### What's the difference bewteen a Front End Designer and Front End Web Developer?”
+
+We expect our Front End Designers to have design as well as HTML, CSS and some JavaScript experience. They should feel comfortable brainstorming and developing design options to meet user needs; developing wireframes or HTML/CSS prototypes to test their design ideas; and quickly iterating on feedback. They should also feel comfortable working as a front end developer on projects, implementing acc ...
```
> We expect our Front End Designers to have design as well

Perhaps clarify "Front End Designer to have experience working with visual design and with visual designers"

What I'm trying to get at is that you don't have to be a Phototoshop, Sketch, or Illustrator master, however, you should know how to stumble through the software and "do" visual design basics. We're suggesting that you have a strong eye for design, even if you're not the type of person who can sit down and create custom illustrations.

If we could refine with that in mind, I think we're on the right track and help eliminate some further confusion.

russu commented on pages/who-we-are-hiring.md:
```diff
@@ -18,6 +18,12 @@ We're actively hiring for the following positions:
+
+We expect our Front End Designers to have design as well as HTML, CSS and some JavaScript experience. They should feel comfortable brainstorming and developing design options to meet user needs; developing wireframes or HTML/CSS prototypes to test their design ideas; and quickly iterating on feedback. They should also feel comfortable working as a front end developer on projects, implementing acc ...
+
+By contrast, our Front End Web Developers are not expected to have design experience. Instead, they are expected to have much more experience than our Front End Designers with JavaScript, including strict mode compliance, modularization techniques and tools, and frameworks and libraries. They are expected to be fully comfortable writing vanilla JavaScript, working with APIs, and making front end  ...
```
> "By contrast, our Front End Web Developers are not expected to have design experience."

I think it may be important to change this to "visual design" versus "design" because we have so many designs, and we should get used to have descriptors for them.

maya replied:
Agree, also bc frontend developers are designing how they write their code, so it's not accurate.

meiqimichelle committed:
Adjust wording based on feedback

- Removes 'web' from developer position name for consistency
- Clarifies 'visual' design
- Attempts to clarify the type of design sense we're looking for (ie, desire for design sensibility and design basics, but not like custom illustrations)

meiqimichelle commented:
@shawnbot @maya @msecret I made adjustments based on your feedback. Review again?

fureigh commented on pages/who-we-are-hiring.md:
```diff
@@ -18,6 +18,12 @@ We're actively hiring for the following positions:
 
 {% include openpositions.html %}
 
+### What's the difference bewteen a Front End Designer and Front End Developer?”
```
'bewteen' --> 'between'

meiqimichelle replied:
Thanks @fureigh ! Fixed with https://github.com/18F/joining-18f/commit/8fa26bb2c32d35771bf5785b90fa886ff4e6140a

meiqimichelle committed:
Fix typo

meiqimichelle commented:
@rooneywp I think this is close to ready. What is Team Talent's process for getting new words on the Join Us site?

meiqimichelle commented:
Hrm. Also I see the hiring page has been updated so it doesn't have front end designer or developer anymore so this adjustment might be OBE. @rooneywp I'll catch you offline to learn the latest.

fureigh commented:
@meiqimichelle Yeah, there was a quick temporary shift in the position descriptions. Your adjustment should still be relevant in the near future, though. Looking forward to incorporating it.

fureigh commented:
@meiqimichelle @rooneywp Should this be incorporated now or does the new content cover it? If neither is the case, it may be relevant to #99.

pixelwake commented:
@fureigh @meiqimichelle I think we should have this covered. The final output was an updated performance profile which we will incorporate as part of #99

meiqimichelle commented:
@rooneywp @fureigh Yep, this issue is OBE. The idea that started here became the current draft performance profile for the front end designer position. I'll cc myself on #99 in case anything comes up there, and close this issue.
